src/resultsbottomuppage.cpp

```diff
@@ -95,6 +96,25 @@ ResultsBottomUpPage::ResultsBottomUpPage(FilterAndZoomStack* filterStack, PerfPa
                 }
             }
         });
+
+    struct AggregationType
+    {
+        QString name;
+        Settings::CostAggregation aggregation;
+    };
+
+    for (const auto& aggregationType :
+         QList<AggregationType>({{QLatin1String("Symbol"), Settings::CostAggregation::BySymbol},
```

Comment:
don't use a QList (required memory allocation), use an initializer list directly

also, use tr() to get translated strings